infraohjelmointi_api/migrations/0024_projecthashtag_remove_project_hashtags_and_more.py

```diff
@@ -0,0 +1,80 @@
+# Generated by Django 4.1.3 on 2023-01-18 10:59
+
```

Comment:
Rename migration file to more describable name. 0024_redefine_project_hashtags_and_create_hashtags_table.py

Reply:
Done